po/extra/pt_BR.po

```diff
@@ -0,0 +1,456 @@
+msgid "Reco"
+msgstr "Reco"
+
+#: data/reco.appdata.xml.in:9 data/reco.desktop.in:5
+msgid "Record talks to remember the contents later"
+msgstr "Gravar conversas para recordar o conteúdo mais tarde"
+
+#: data/reco.appdata.xml.in:11
+msgid "An audio recording app that helps you recall things talked later."
+msgstr ""
+"Um aplicativo de gravação áudio que o ajuda a recordar as coisas faladas "
+"mais tarde."
+
+#: data/reco.appdata.xml.in:14
+msgid "Features include:"
+msgstr "Funções incluídas:"
+
+#: data/reco.appdata.xml.in:16
+msgid ""
+"Recording sounds from both your microphone and system at the same time. This "
+"would be useful when you record conversations with chat apps or stream "
+"videos on the Internet"
+msgstr ""
+"Gravação de sons tanto do seu microfone como do sistema ao mesmo tempo. Isto "
+"será útil quando se grava conversas com aplicações de chat ou transmissões "
+"de vídeos na Internet"
+
+#: data/reco.appdata.xml.in:17
+msgid ""
+"Saving in some commonly used formats. It supports AAC, FLAC, MP3, Ogg "
+"Vorbis, Opus, and WAV"
+msgstr ""
+"Grave em alguns formatos normalmente utilizados. Suporta AAC, FLAC, MP3, "
+"Ogg Vorbis, Opus, e WAV"
+
+#: data/reco.appdata.xml.in:18
+msgid ""
+"Timed recording. You can set a delay before recording up to 15 seconds, and "
+"the length of recording up to 600 seconds"
+msgstr ""
+"Gravação cronometrada. Pode definir um atraso antes da gravação até 15 "
+"segundos e a duração da gravação até 600 segundos"
+
+#: data/reco.appdata.xml.in:19
+msgid ""
+"Autosaving or always-ask-where-to-save workflow. You can select whether the "
+"app saves recordings automatically or you choose a destination each time on "
+"saving them"
+msgstr ""
+"Guardar automaticamente ou perguntar sempre onde guardar o fluxo de "
+"trabalho. Pode selecionar se a aplicação salva as gravações automaticamente "
+"ou se escolhe um destino cada vez que as salva"
+
+#: data/reco.appdata.xml.in:20
+msgid ""
+"Saving recording when the app quits. Even if you happen to quit the app "
+"while recording, the recording is saved automatically or a FileChooserDialog "
+"is shown depends on your saving setting"
+msgstr ""
+"Salvar a gravação quando a aplicação deixa de funcionar. Mesmo que saia da "
+"aplicação durante a gravação, a gravação é guardada automaticamente ou é "
+"mostrada uma Janela de escolha de arquivos dependendo da sua definição de "
+"gravação"
+
+#: data/reco.appdata.xml.in:25
+msgid "Welcome screen that allows you to set recording configuration"
+msgstr "Tela de boas-vindas que lhe permite definir a configuração da gravação"
+
+#: data/reco.appdata.xml.in:29
+msgid "Recording screen"
+msgstr "Tela de gravação"
+
+#: data/reco.appdata.xml.in:33
+msgid "Recording is successfully finished"
+msgstr "A gravação foi concluída com sucesso"
+
+#: data/reco.appdata.xml.in:72
+#, fuzzy
+msgid "Update Portuguese translation (thanks to @hugok79)"
+msgstr "Tradução portuguesa adicionada (obrigado a @hugok79)"
```

Comment:
* Remove the "fuzzy" flag
* The word "Update" seems to be translated as "Add" maybe? (as far as I checked this translation using Google Translate)